mods/mkrcraft/src/main/kotlin/pw/mkr/craft/ServerInit.kt

```diff
@@ -0,0 +1,71 @@
+package pw.mkr.craft
+
+import net.fabricmc.api.DedicatedServerModInitializer
+import net.minecraft.entity.player.PlayerEntity
+import net.minecraft.server.network.ServerPlayerEntity
+import net.minecraft.text.LiteralText
+import net.minecraft.world.GameMode
+import pw.mkr.craft.blocks.binding.BindingBlock
+import pw.mkr.craft.events.chunkentry.ChunkEntryEvent
+import pw.mkr.craft.events.chunkentry.ChunkEntryListener
+import pw.mkr.craft.models.Binding
+import pw.mkr.craft.models.Chunk
+import pw.mkr.craft.blocks.portals.PortalBlock
+import pw.mkr.craft.utils.StoreManager
+
+@Suppress("Unused")
+object ServerInit : DedicatedServerModInitializer {
+    const val MOD_ID = "mkrcraft"
+
+    override fun onInitializeServer() {
+        StoreManager.init()
+        BindingBlock.register()
+        PortalBlock.register()
+
+        ChunkEntryEvent.listen(object : ChunkEntryListener {
```

Comment:
Can we move this implementation to `BindingBlock.register`, and have `PortalBlock` extend `BindingBlock` (because it inherits all of the same mechanics)?

Edit: I realise it's not straightforward to extend companion object? Please investigate how to do that.